server/json_client.go

```diff
@@ -0,0 +1,105 @@
+package server
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/mysterium/node/identity"
+	"net/http"
+	"net/url"
+)
+
+const (
+	mysteriumAgentName       = "goclient-v0.1"
+	authenticationHeaderName = "Authorization"
+	authenticationSchemaName = "Signature"
+)
+
+var noSignature = identity.SignatureBytes(nil)
+
+type HttpClient interface {
+	DoGet(path string, values url.Values) (*http.Response, error)
```

Comment:
Is it worth prefixing the methods with "Do"?

Reply:
Well obj.Get usually says get something from object. I see a better intent expression here. as GET POST etc. are actions and DoAction provides better readability. Ofcourse if it's confusing it can be renamed

Reply:
Agree with @ignasbernotas - `Get` is a verb itself and since this is not Java, getters like `GetAmounts` shouldn't be very common to confuse with.

Reply:
Will rename it to Get Post and SignedPost